lib/adequate_crypto_address/base.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module AdequateCryptoAddress
+  class Base
+    attr_reader :address
+
+    def initialize(address_string)
+      @address = address_string
+    end
+
+    def valid?(address)
+      valid_prefix?
```

Comment:
We're not checking `valid_length?` here -- `valid_prefix? && valid_length?`

Reply:
I didn't include that here bc I wasn't checking `valid_length?` for every class/coin ://